subscription_manager.go

```diff
@@ -53,3 +63,8 @@ func (s *subscriptionManager) notifySubscribers(hash lntypes.Hash, setID [32]byt
 func (s *subscriptionManager) hasSubscribers() bool {
 	return len(s.acceptSubscriptions) > 0
 }
+
+func (s *subscriptionManager) isWaitingForReconnection() bool {
```

Comment:
Maybe this and `hasSubscribers` can be merged into a single function? Perhaps name it something more abstract like `hasRecentSubscribers()`?

Reply:
Yep indeed